tests/e2e.test.js

```diff
@@ -127,6 +127,16 @@ describe('External links', () => {
       expect(page).toClick(link.getProperty('innerText')),
     );
   });
+
+  it('is possible to click outdated icon link', async () => {
+    const reportIcons = await page.$$('a.report__icon');
+    expect(page).toClick(
+      await getAttribute(
+        reportIcons[Math.floor(Math.random() * reportIcons.length)],
+        'href',
+      ),
+    );
```

Comment:
Also this, should be in another PR.

Reply:
no, this is supposed to be here. It's "compact layout and report outdated icon" together.